### PR TITLE
Add logging package makefile install and uninstall

### LINKED ISSUE #14: Migrate logging service

### PR DESCRIPTION
**Pull request by nehayward**
Follows existing pattern and adds logging helm package for easy install and uninstall.

closes #14

### CODE REVIEW

File: addons/logging/Makefile [hunk 1]
```diff
@@ -1,11 +1,11 @@
 install:
-	helm install "TODO" \
+	helm install ./fluentd-cloudwatch-0.0.1.tgz \
 		--name "fluentd" \
 		--set aws.access_key="$(AWS_ACCESS_KEY_ID)" \
```
Comment (nehayward) [inline addons/logging/Makefile]: Based on previous discussion, should I add ifndef before install commands for env variables?

Comment (nehayward) [reply addons/logging/Makefile]: Actually it'd only really be applicable for install, so we could follow the previous pattern:
```
install: check
...

check:
ifndef: ... (check for expected ENV)
```

Comment (zpatrick) [reply addons/logging/Makefile]: Yeah, I think that's probably the best pattern to adopt for these types of scenarios

Comment (zpatrick) [reply addons/logging/Makefile]: also, you may want to consider removing that `terraform output` command in the root `Makefile`; it's nice UX but inconsistent with how we do other required env vars.

**Pull request merged**